internetarchive/cli/ia_download.py

```diff
@@ -167,53 +183,59 @@ def main(argv, session: ArchiveSession) -> None:
 
     errors = []
     for i, identifier in enumerate(ids):
-        try:
-            identifier = identifier.strip()
-        except AttributeError:
-            identifier = identifier.get('identifier')
-        if total_ids > 1:
-            item_index = f'{i + 1}/{total_ids}'
+        if end_idx != None and end_idx == i:
```

Comment:
None is a singleton. Use `is not None` for the comparison. In this case `end_index == i` would also suffice as condition.